articles/142_idl.md

```diff
@@ -30,6 +30,114 @@ Further it describes how the interfaces are used to generate code in C, C++ and
   <b>TODO:</b> All of this
 </div>
 
+ROS 2 supports a subset of the [OMG IDL 4.2 specification](https://www.omg.org/spec/IDL/4.2).
+
+### Basic Types
+
+#### 7.4.1.4.4.2.1 Integer Types
+
+| IDL type       | Value range        |
+| -------------- | ------------------ |
+| short          | -2^15 ... 2^15 - 1 |
+| unsigned short | 0 ... 2^16 - 1     |
+| int            | -2^31 ... 2^31 - 1 |
```

Comment:
In 4.2 it looks like the IDL type with this range is `long`. I don't see a type called `int`.

Reply:
Correct, fixed in 4cdcc0f5879d921b0128c667ba794d94f66e9ad2.